cmd/acr/login.go

```diff
@@ -9,14 +9,15 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"os"
 	"strings"
 
-	dockerAuth "github.com/Azure/acr-cli/auth/docker"
+	orasgoAuth "github.com/Azure/acr-cli/auth/oras-go"
```

Comment:
Please try making the package name alias all in lower letters.

Reply:
changed it to `oauth`.

Reply:
removed the alias.